sdk/profiles/hybrid20200901/resourcemanager/authorization/armauthorization/autorest.md

```diff
@@ -0,0 +1,17 @@
+### AutoRest Configuration
+
+> see https://aka.ms/autorest
+
+``` yaml
+azure-arm: true
+require:
+- https://github.com/Azure/azure-rest-api-specs/blob/53b1affe357b3bfbb53721d0a2002382a046d3b0/specification/authorization/resource-manager/readme.md
+- https://github.com/Azure/azure-rest-api-specs/blob/53b1affe357b3bfbb53721d0a2002382a046d3b0/specification/authorization/resource-manager/readme.go.md
+license-header: MICROSOFT_MIT_NO_VERSION
+module-version: 0.1.0
```

Comment:
This, and `module-name`, should be removed as the sub packages aren't modules.

Reply:
For current codegen, without module-version configured, arm package could not be generated successfully.

Reply:
What was the error?  Is this somethin we need to fix in the code generator?

Reply:
Current codegen will always generate `moduleVersion` and `moduleName` constants ([code](https://github.com/Azure/autorest.go/blob/2fe13f887fd5a7def9a77590d294545239ec8433/src/generator/constants.ts#L34). And these two constants will be used when build pipeline ([code](https://github.com/Azure/autorest.go/blob/2fe13f887fd5a7def9a77590d294545239ec8433/src/generator/operations.ts#L214)). If we need to change codegen, we have to decide when it is a sub-package, what we should do for the pipeline building.

Reply:
Got it, thanks for clarifying.

Reply:
@jhendrixMSFT Regarding the solution you and Jeff propose, do we need to change code generator to support generate the sub package, or just like this PR, do some manual work or maybe could use directive to change the code?

Reply:
I suspect the solution will be a combination of small changes to the code generator and some autorest transforms.